src/pykeen/models/unimodal/node_piece.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+
+"""A wrapper which combines an interaction function with NodePiece entity representations."""
+
+import logging
+from typing import Any, Callable, ClassVar, Mapping, Optional, Sequence, Union
+
+import torch
+from class_resolver import Hint, HintOrType
+from torch import nn
+
+from ..nbase import ERModel
+from ...constants import DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE
+from ...nn.emb import EmbeddingSpecification, NodePieceRepresentation
+from ...nn.modules import DistMultInteraction, Interaction
+from ...triples.triples_factory import CoreTriplesFactory
+
+__all__ = [
+    "NodePiece",
+]
+
+logger = logging.getLogger(__name__)
+
+
+class _ConcatMLP(nn.Sequential):
+    """A 2-layer MLP with ReLU activation and dropout applied to the concatenation of token representations.
+
+    This is for conveniently choosing a configuration similar to the paper. For more complex aggregation mechanisms,
+    pass an arbitrary callable instead.
+
+    .. seealso:: https://github.com/migalkin/NodePiece/blob/d731c9990/lp_rp/pykeen105/nodepiece_rotate.py#L57-L65
+    """
+
+    def __init__(
+        self,
+        num_tokens: int,
+        embedding_dim: int,
+        dropout: float = 0.1,
+        ratio: Union[int, float] = 2,
```

Comment:
@cthoyt this parameter is not exposed via the `NodePiece`model, and the class is not intended to be used outside. So maybe we can drop the parameter again?

Reply:
That's true, but I like the idea of it being changable/documented through the kwargs